jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -5354,4 +5354,41 @@ public IntStream codePoints() {
 		return value;
 	}
 /*[ENDIF] Java10 */	
+
+/*[IF Java11]*/
+	/**
+	 * Returns a string object containing the character (Unicode code point)
+	 * specified.
+	 * 
+	 * @param codePoint
+	 *            a Unicode code point.
+	 * @return a string containing the character (Unicode code point) supplied.
+	 * @throws IllegalArgumentException
+	 *             if the codePoint is not a valid Unicode code point.
+	 * @since 11
+	 */
+	static String valueOfCodePoint(int codePoint) {
+		String string;
+		if ((codePoint < Character.MIN_CODE_POINT) || (codePoint > Character.MAX_CODE_POINT)) {
+			/*[MSG "K0800", "Invalid Unicode code point"]*/
+			throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K0800"));	//$NON-NLS-1$
+		} else if (codePoint <= 255) {
+			if (enableCompression) {
+				string = new String(compressedAsciiTable[codePoint], 0, 1, true);
+			} else {
+				string = new String(decompressedAsciiTable[codePoint], 0, 1, false);
```

Comment:
Would it be possible to use:

https://github.com/JasonFengJ9/openj9/blob/c864cc795b78b3b1c573e8bc5684c7f9f413c6af/jcl/src/java.base/share/classes/java/lang/String.java#L402

Constructor here instead? The one currently being used will have to perform many of the checks again. Similarly for all other assignments to `string` in this method.

Reply:
`String(byteArray, coder)` works as well, will update the PR accordingly.